src/gordon_gcp/plugins/enricher.py

```diff
@@ -57,37 +58,52 @@ class GCEEnricherBuilder:
         kwargs (dict): Additional keyword arguments to pass to the
             enricher.
     """
+    VALIDATORS = [
+        gordon_gcp.enricher.GCEEnricherBuilder._validate_keyfile,
+        gordon_gcp.enricher.GCEEnricherBuilder._validate_dns_zone,
+        gordon_gcp.enricher.GCEEnricherBuilder._validate_retries
```

Comment:
Do these need to be fully qualified here?

Reply:
They do.  You can't have a class variable refer to static methods without the class name, and that isn't defined yet during the class definition.